web/react/components/about_build_modal.jsx

```diff
@@ -21,29 +21,38 @@ export default class AboutBuildModal extends React.Component {
 
         let title = (
             <FormattedMessage
-                id='about.teamEdtion'
-                defaultMessage='Team Edition'
+                id='about.teamEdtiont0'
```

Comment:
Mind fixing teamEdtion by changing it to teamEdition? I don't want to start another RECIEVE thing again :smile: